src/main/java/seedu/address/logic/Messages.java

```diff
@@ -75,7 +75,7 @@ public static String format(Transaction transaction) {
         final StringBuilder builder = new StringBuilder();
         builder.append(transaction.getDescription())
                 .append("; Amount: ")
-                .append(String.format(".2f", transaction.getAmount()))
+                .append(transaction.getAmount())
```

Comment:
If I try to add:
```
Transaction 1: Amount: 100
Transaction 2: Amount: 100.1
Transaction 3: Amount: 100.11
```
it will show the transactions like above, without the trailing zeroes. Do we want to force the amounts to appear with 2d.p? If not, I think this is fine

Reply:
Initially I changed this line because the % sign was missing so the String.format() did not work properly, now I changed it so it's by default 2 d.p. already. This only applies to the `addt` message though, on the personCard and transactionCard there are no trailing zeroes.